Source/WebCore/css/CSSSelector.cpp

```diff
@@ -182,6 +184,7 @@ SelectorSpecificity simpleSelectorSpecificity(const CSSSelector& simpleSelector)
             return maxSpecificity(simpleSelector.selectorList());
         return SelectorSpecificityIncrement::ClassC;
     case CSSSelector::Match::Unknown:
+    case CSSSelector::Match::UnknownToSerialize:
```

Comment:
Does CSSSelector::Match::Unknown ever actually occur? That asserts suggest it doesn't.

Maybe `CSSSelector::Match::ForgivingUnknown` or similar?

Reply:
Changed to `ForgivingUnknown` and `ForgivingUnknownNestContaining`.